### ISSUE TITLE
Problems with .WithColumnDescription(string) option

### ISSUE THREAD

**Issue by kroliczek94** [fluentmigrator/fluentmigrator#971]
I am creating a new table in Fluent Migrator which is located in some schema ('central'). It contains a column with int type, which is mapping some enumerator. This is definition of this column.
`Create.Table("synchronizations").InSchema("central")
.WithColumn("state").AsCustom("int").NotNullable().WithColumnDescription("0-NonSynced, 1-Synced, 2-Syncing, 3-SyncingError")`

Generated SQL code is 
`COMMENT ON COLUMN "central"."public"."synchronizations"."state" IS '0-NonSynced, 1-Synced, 2-Syncing, 3-SyncingError';;`

Table "synchronizations" is belong to "central" schema, not to default "public" schema. This is the reason of exception: " Cross-database references are not implemented".

Why 'public' schema is added to this table name? How can I get around this bug?

**Comment by fubar-coder** [comment]
The current "workaround" is to copy the source of the `PostgresGenerator` and `PostgresDescriptionGenerator` into your project, rename them to `CustomPostgresGenerator` and `CustomPostgresDescriptionGenerator` and register the generator as follows:

```csharp
services.AddScoped<PostgresGenerator, CustomPostgresGenerator>();
```

The contents of the `CustomPostgresDescriptionGenerator` should be changed to:

```csharp
using FluentMigrator.Runner.Generators.Generic;

namespace FluentMigrator.Runner.Generators.Postgres
{
    /// <summary>
    /// almost copied from OracleDescriptionGenerator,
    /// modified for escaping table description
    /// </summary>
    public class CustomPostgresDescriptionGenerator : GenericDescriptionGenerator
    {
        public PostgresDescriptionGenerator()
        {
            Quoter = new PostgresQuoter();
        }

        protected IQuoter Quoter { get; }

        #region Constants

        private const string TableDescriptionTemplate = "COMMENT ON TABLE {0} IS '{1}';";
        private const string ColumnDescriptionTemplate = "COMMENT ON COLUMN {0}.{1} IS '{2}';";

        #endregion

        protected override string GenerateTableDescription(
            string schemaName, string tableName, string tableDescription)
        {
            if (string.IsNullOrEmpty(tableDescription))
                return string.Empty;

            return string.Format(TableDescriptionTemplate, Quoter.QuoteTableName(tableName, schemaName), tableDescription.Replace("'", "''"));
        }

        protected override string GenerateColumnDescription(
            string schemaName, string tableName, string columnName, string columnDescription)
        {
            if (string.IsNullOrEmpty(columnDescription))
                return string.Empty;

            return string.Format(
                ColumnDescriptionTemplate,
                Quoter.QuoteTableName(tableName, schemaName),
                Quoter.QuoteColumnName(columnName),
                columnDescription.Replace("'", "''"));
        }
    }
}
```

The `CustomPostgresGenerator` constructor should be changed to:

```csharp
        public CustomPostgresGenerator(
            [NotNull] PostgresQuoter quoter,
            [NotNull] IOptions<GeneratorOptions> generatorOptions)
            : base(new PostgresColumn(), quoter, new CustomPostgresDescriptionGenerator(), generatorOptions)
        {
        }
```

This should unblock you.

**Comment by fubar-coder** [comment]
Fixed by PR #999
